Sources/OAuth2Client/NXOAuth2AccountStore.m

```diff
@@ -545,12 +547,8 @@ - (void)oauthClientDidRefreshAccessToken:(NXOAuth2Client *)client
     }
     
     foundAccount.accessToken = client.accessToken;
-    NSDictionary *userInfo = [NSDictionary dictionaryWithObject: foundAccount
-                                                         forKey: NXOAuth2AccountStoreNewAccountUserInfoKey];
-    
-    [[NSNotificationCenter defaultCenter] postNotificationName:NXOAuth2AccountStoreAccountsDidChangeNotification
-                                                        object:self
-                                                      userInfo:userInfo];
+    [[NSNotificationCenter defaultCenter] postNotificationName:NXOAuth2AccountDidChangeAccessTokenNotification
```

Comment:
Sorry, I was attempting to make the calls consistent. The access token has changed, so I used this notification type. I understand the account has changed also.